src/components/FoilOverlay.js

```diff
@@ -1,9 +1,8 @@
 /* eslint-disable react/prop-types */
```

Comment:
These are the linter errors that should be fixed rather than disabled :slightly_smiling_face: